README.md

```diff
@@ -253,15 +254,16 @@ Providing examples or tutorial of how to use your code can be an extremely helpf
 ### Welcoming Contributions
 In your `README.md` or a separate `CONTRIBUTING.md` document, clearly specify the types of contributions that are welcome. This could include code contributions, documentation improvements, bug reports, feature requests, etc.
 
-## Project Visualisation
+### Project Visualization
 
-When applicable it is helpful to visualise the project you are working on, especially when working on pipeline projects. Visualising key components of the process, and how different stages of the project fit together can help in the design and communication about the project, especially in new projects where there are many moving parts, where it is imparative to have team members be clear on the vision of the project. Diagrams can be a very effective way of making sure the team is on the same page. In addition it is a fast way of communicating how the project looks and what it will involve with a wide range of people with varying backgrounds. Visualisations also help with onboarding, since new team members can quickly get a high level understanding of where the project is at and how the various pieces fit together.
+When applicable it is helpful to visualize the project you are working on, especially when working on pipeline projects, databse design, or composable programing. Visualizing key components of the process, and how different stages of the project fit together can help in the design and communication about the project, especially in new projects where there are many moving parts, where it is imperative to have team members be clear on the vision of the project. Diagrams can be a very effective way of making sure the team is on the same page. In addition it is a fast way of communicating how the project looks and what it will involve with a wide range of people with varying backgrounds. Visualisations also help with onboarding, since new team members can quickly get a high level understanding of where the project is at and how the various pieces fit together.
```

Comment:
Reads as repetitive - shorten down, maybe something like:

Visualizing key aspects of a project’s structure (e.g. data pipelines, database schemas, or function chains), with diagrams, helps to provide a clear high-level overview to a project, and assists in discussing modifications or improvements to the code-base.